features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -513,7 +513,7 @@ Feature: Gift Exchange Challenge
         And the email should contain "Additional Tags:"
         And the email should contain "fic"
         And the email should contain "art"
-        And the email should contain "Character:"
+        And the email should contain "Characters:"
```

Comment:
Snuck in from the challenge assignment internationalisation, I think, but odd that we didn't get a test failure in that because of it. I'll check.